spec/support/features/session_helper.rb

```diff
@@ -79,7 +79,8 @@ def confirm_last_user
       User.last.update(
         confirmation_token: @raw_confirmation_token, confirmation_sent_at: Time.current
       )
-      visit "/users/confirmation?confirmation_token=#{@raw_confirmation_token}"
+      visit "/sign_up/create_password?confirmation_token=#{@raw_confirmation_token}"
+      # visit user_confirmation_url(confirmation_token: @raw_confirmation_token)
```

Comment:
good idea! Just tested out with `_path` instead and the failure is still there.  

```
Failures:

  1) Email confirmation during sign up confirmed user is signed out and tries to confirm again redirects to sign in page with message that user is already confirmed
     Failure/Error:
       expect(page).to have_content(
         t('devise.confirmations.already_confirmed', action: 'Please sign in.')
       )

       expected to find text "Your email address has already been confirmed. Please sign in." in "An official website of the United States governmentYou have confirmed your email address123456Create a strong passwordYour password must be at least 8 characters long. It can be a mix of alphanumeric and special characters and individual words in a phrase. Create a password that’s secure, but still easy for you to remember. And don’t use the same one you use with your other online accounts, such as bank, email, and social media. * PasswordPassword strength: ...U.S. General Services AdministrationHelpContactPrivacy Policy"
     # ./spec/features/visitors/email_confirmation_spec.rb:128:in `block (3 levels) in <top (required)>'

  2) Email confirmation during sign up confirmed user is signed in and tries to confirm again redirects the user to the profile
     Failure/Error: expect(current_url).to eq profile_url

       expected: "http://www.example.com/profile"
            got: "http://www.example.com/sign_up/create_password?confirmation_token[]=PhtRwDMu7JjCh48txasH&confirmation_token[]=66217b7df8ed11e9fe894ce539dce8a710fd8c666c9069b76bac951bb769e70d"

       (compared using ==)
     # ./spec/features/visitors/email_confirmation_spec.rb:117:in `block (3 levels) in <top (required)>'

Finished in 12.31 seconds (files took 8.03 seconds to load)
12 examples, 2 failures

Failed examples:

rspec ./spec/features/visitors/email_confirmation_spec.rb:122 # Email confirmation during sign up confirmed user is signed out and tries to confirm again redirects to sign in page with message that user is already confirmed
rspec ./spec/features/visitors/email_confirmation_spec.rb:112 # Email confirmation during sign up confirmed user is signed in and tries to confirm again redirects the user to the profile
```